observability/otel/otel-collector/otel-collector-extensions.adoc

```diff
@@ -446,18 +445,38 @@ include::snippets/technology-preview.adoc[]
         protocols:
```

Comment:
A sentence explaining the YAML would be useful as it seems a bit dropped in to the module.

Reply:
It's just an example configuration to have something users can copy, paste, and test.

Reply:
NP. The heading might be enough to explain the purpose of the YAML. Feel free to send this for merging.

Reply:
@max-cx , are you OK with this to move forward. I'm happy with the updates.